src/sentry/replays/endpoints/project_replay_clicks_index.py

```diff
@@ -41,14 +47,44 @@
 REFERRER = "replays.query.query_replay_clicks_dataset"
 
 
+class ReplayClickResponseData(TypedDict, total=False):
+    node_id: int
+    timestamp: str
+
+
+class ReplayClickResponse(TypedDict):
+    data: list[ReplayClickResponseData]
```

Comment:
With `total=False`, did you want to say that the `data` list can be empty, or that each item may have either `node_id` or `timestamp` missing?